CHANGELOG.md

```diff
@@ -4,9 +4,15 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [8.0.1]
+## [9.0.0]
+
+### Changed
+- A job's `processing_times` field is now set to `null` rather than `[]` if there was an error while calculating processing times. As before, an error while calculating processing times causes the job to fail.
```

Comment:
Ah yeah, I think I worded this confusingly, partially because I was attempting to summarize some Mattermost discussion with Joe, but I don't think it comes through here. Let me workshop it a bit.

Reply:
Simplified the Changelog, API spec, and code as a result of this feedback, thanks!